CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/MediaPlayer` layout issues for Cobalt, Carbon, Copper, Electro, Titanium skins
 - `agate/Picker` and `agate/RangePicker` to match latest design for Silicon skin
 - `agate/Scroller` to update scrollButtons state on initial render
+- `agate/ThemeDecorator` so `bg` class works only at the apps' root node not at a child node of the app's root node
```

Comment:
Would you please rewrite the changelog to the same format as other Fixed section? 
I'd try to read but I can't understand.
(Fixed) `agate/ThemeDecorator` so `bg` class works only at the... ? 
-> (Fixed) `agate/ThemeDecorator` to ACTION...

Reply:
I have pushed a fix for the CHANGELOG. Please let me know if it's better now.